### PR TITLE
Onos classic configurability

### PR DESCRIPTION
**Pull request by RandyLevensalor**
Adding option image pull secret to stateful set to support pulling
from a secure repo.

Adding an optional file mode for mounting volumes in onos-config-loader

### CODE REVIEW

**Comment by RandyLevensalor** [comment]
I have a current ONF/cla.  How do I link it with my GitHub account?

**Comment by RandyLevensalor** [comment]
@adibrastegarnia Can you please review this?

**Review by pierventre — APPROVED** [review]

**Review by amolonf — APPROVED** [review]

File: onos-classic/templates/onos-config-loader-script.yaml [hunk 1]
```diff
@@ -38,6 +38,8 @@ data:
     NETCFG="/opt/configs/netcfg.json"
     RECONCILE_MODE="{{ .Values.config.reconcile_mode }}";
 
+    # run in tmp instead of root
+    cd /tmp
```
Comment (Andrea-Campanella) [inline onos-classic/templates/onos-config-loader-script.yaml:42]: Hi Randy, thanks for the patch! is this change to /tmp because of security concerns ?

Comment (RandyLevensalor) [reply onos-classic/templates/onos-config-loader-script.yaml:42]: Yes.  As a best practice, we do everything possible to not run pods as a privileged user.

When the script is run from the default directory, the script tires to create files in a directory that it doesn't have permission to write.

With this change to /tmp, any user can run the script successfully.

File: onos-classic/Chart.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 ---
 apiVersion: v1
 name: onos-classic
-version: 0.1.30
+version: 0.1.33
```
[the remark this answers is not in the document]
Comment (RandyLevensalor) [reply onos-classic/Chart.yaml]: I bumped the version a few times during dev and missed rolling it back.  It's now rolled back to 1.31.  Thanks for the review.

**Review by charlesmcchan — APPROVED** [review]

**Comment by amolonf** [comment]
retest

**Comment by amolonf** [comment]
retest

**Comment by pierventre** [comment]
retest

**Comment by SeanCondon** [comment]
retest this please

**Comment by SeanCondon** [comment]
you've got to say it nicely

**Pull request merged**